go/vt/binlog/keyrange_filter.go

```diff
@@ -31,7 +31,7 @@ import (
 // in the transaction match the specified keyrange. The resulting function can be
 // passed into the Streamer: bls.Stream(file, pos, sendTransaction) ->
 // bls.Stream(file, pos, KeyRangeFilterFunc(keyrange, sendTransaction))
-func KeyRangeFilterFunc(keyrange *topodatapb.KeyRange, callback func(*binlogdatapb.BinlogTransaction) error) sendTransactionFunc {
+func KeyRangeFilterFunc(keyrange *topodatapb.KeyRange, callback func(*binlogdatapb.BinlogTransaction) error) sendTransactionFunc { // nolint:revive
```

Comment:
curious what the linter complain here was

Reply:
```
go/vt/binlog/keyrange_filter.go:34:110: unexported-return: exported func KeyRangeFilterFunc returns unexported type binlog.sendTransactionFunc, which can be annoying to use (revive)
```